### PR TITLE
fix inertia for ur5

### PR DESCRIPTION
**Pull request by k-okada**

### CODE REVIEW

File: ur_description/urdf/ur5.urdf.xacro [hunk 1]
```diff
@@ -303,14 +303,7 @@
...
 
-    <link name="${prefix}ee_link">
-      <collision>
-        <geometry>
-          <box size="0.01 0.01 0.01"/>
-        </geometry>
-        <origin rpy="0 0 0" xyz="-0.01 0 0"/>
-      </collision>
-    </link>
+    <link name="${prefix}ee_link" />
```
Comment (fmessmer) [inline ur_description/urdf/ur5.urdf.xacro]: Please do not remove the collision geometry...it has been added on purpose...see #122

**Comment by fmessmer** [comment]
Where did you take the valus for radius and the lenght from?
Could you update the models for UR3 and UR10 as well?

File: ur_description/urdf/ur5.urdf.xacro (hunk 1 above)
Comment (k-okada) [inline ur_description/urdf/ur5.urdf.xacro]: sorry for super late, I have  fixed this

**Comment by k-okada** [comment]
@ipa-fxm Where did you take the values for radius and the lenght from?

I just determine the value by watching  model shape and collision shape in rviz, I do not have real robot so may be someone in this repo may know better parameters

**Comment by gavanderhoorn** [comment]
I'm hesitant to merge this PR as it changes properties that the Gazebo simulation depends on.

These kind of inertia values are always approximations, but in this case @k-okada 'admits' that he tweaked them by "watching model shape and collision shape in rviz". That seems to me like one step up on the 'guestimation ladder' from what @ipa-fxm did.

I'll leave this open for now, but it'd be really nice if we could somehow quantify the improvement the proposed changes are supposed to bring.

**Comment by gavanderhoorn** [comment]
The values proposed by @k-okada are quite different (smaller) from what we have currently in the xacro macros.

It'd be nice if we could get some additional eyes on this during `wrid18`.

**Comment by henryponti** [comment]
It looks like the radius of the wrist links are incorrect in the current xacro file:

  229 <xacro:cylinder_inertial radius="0.6" length="0.12" mass="${wrist_1_mass}">
  262 <xacro:cylinder_inertial radius="0.6" length="0.12" mass="${wrist_2_mass}">
  295 <xacro:cylinder_inertial radius="0.6" length="0.12" mass="${wrist_3_mass}">

I believe they should be 0.06 instead of 0.6:

```
diff --git a/ur_description/urdf/ur5.urdf.xacro b/ur_description/urdf/ur5.urdf.xacro
index 941ea0c9..a956d98f 100644
--- a/ur_description/urdf/ur5.urdf.xacro
+++ b/ur_description/urdf/ur5.urdf.xacro
@@ -226,7 +226,7 @@
           <mesh filename="package://ur_description/meshes/ur5/collision/wrist1.stl" />
         </geometry>
       </collision>
-      <xacro:cylinder_inertial radius="0.6" length="0.12" mass="${wrist_1_mass}">
+      <xacro:cylinder_inertial radius="0.06" length="0.12" mass="${wrist_1_mass}">
         <origin xyz="0.0 0.0 0.0" rpy="0 0 0" />
       </xacro:cylinder_inertial>
     </link>
@@ -259,7 +259,7 @@
           <mesh filename="package://ur_description/meshes/ur5/collision/wrist2.stl" />
         </geometry>
       </collision>
-      <xacro:cylinder_inertial radius="0.6" length="0.12" mass="${wrist_2_mass}">
+      <xacro:cylinder_inertial radius="0.06" length="0.12" mass="${wrist_2_mass}">
         <origin xyz="0.0 0.0 0.0" rpy="0 0 0" />
       </xacro:cylinder_inertial>
     </link>
@@ -292,7 +292,7 @@
           <mesh filename="package://ur_description/meshes/ur5/collision/wrist3.stl" />
         </geometry>
       </collision>
-      <xacro:cylinder_inertial radius="0.6" length="0.12" mass="${wrist_3_mass}">
+      <xacro:cylinder_inertial radius="0.06" length="0.12" mass="${wrist_3_mass}">
         <origin xyz="0.0 0.0 0.0" rpy="0 0 0" />
       </xacro:cylinder_inertial>
     </link>

```

**Comment by gavanderhoorn** [comment]
I'm going to close this and consider #426 as the replacement.

**Pull request closed without merging**